recipes/acts/recipe.yaml

```
@@ -0,0 +1,47 @@
+context:
+  name: acts
+  version: "41.1.0"
+
+package:
+  name: ${{ name|lower }}-core
+  version: ${{ version }}
+
+source:
+  git: https://github.com/acts-project/acts.git
+  tag: v${{ version }}
+
+build:
+  script:
+    - cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - cmake -LH build
+    - cmake --build build --parallel="${CPU_COUNT}"
+    - cmake --install build
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
```

```suggestion
  build:
    - ${{ stdlib('c') }}
    - ${{ compiler('cxx') }}
    - cmake
    - make
```

ACTS is a C++ project so you need the C++ compiler metapackage, and you CMake is used but not defined as a `build` requirement.